addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -45,12 +45,14 @@ $-editor-messages-margin-x: 2%;
         opacity: 0.3;
     }
 
-    &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &:empty {
+    &.oe_structure.oe_empty, .oe_structure.oe_empty {
+        /*Making the drop zone displayed on drag small*/
+        &.oe_structure_small > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
+            padding: 12px 0px;
+        }
+        /*Display drop zone with placeholder on empty page*/
+        &:empty:only-child {
```

Comment:
Weak condition with `only-child`. Why not targeting `#wrap.oe_structure.oe_empty`?